blocklist-client/README.md

```diff
@@ -15,11 +15,25 @@ You can run the blocklist client by providing the required environment variables
 - BLOCKLIST_CLIENT_RISK_ANALYSIS__API_URL=`<provider-url>`
 - BLOCKLIST_CLIENT_RISK_ANALYSIS__API_KEY=`<your_api_key>`
 
+   ```bash
+    BLOCKLIST_CLIENT_RISK_ANALYSIS__API_URL=https://your-risk-provider-api.com/ BLOCKLIST_CLIENT_RISK_ANALYSIS__API_KEY=your_api_key BLOCKLIST_CLIENT_RISK_ANALYSIS__USE_SANCTIONS=true ../target/release/blocklist-client 
```

Comment:
Can you change this configuration example to use the enum specification instead of the old `LOCKLIST_CLIENT_RISK_ANALYSIS__USE_SANCTIONS=true`?

Reply:
```suggestion
    BLOCKLIST_CLIENT_RISK_ANALYSIS__API_URL=https://your-risk-provider-api.com/ BLOCKLIST_CLIENT_RISK_ANALYSIS__API_KEY=your_api_key BLOCKLIST_CLIENT_ASSESSMENT__ASSESSMENT_METHOD=sanctions ../target/release/blocklist-client 
```